components/os/component_output.go

```diff
@@ -231,25 +230,25 @@ func init() {
 
 	// Get boot ID (Linux-specific)
 	var err error
-	currentMachineMetadata.BootID, err = pkg_host.GetBootID()
+	currentMachineMetadata.BootID, err = pkghost.GetBootID()
 	if err != nil {
 		log.Logger.Warnw("failed to get boot id", "error", err)
 	}
 
 	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
-	currentMachineMetadata.DmidecodeUUID, err = pkg_host.DmidecodeUUID(ctx)
+	currentMachineMetadata.DmidecodeUUID, err = pkghost.DmidecodeUUID(ctx)
 	cancel()
 	if err != nil {
 		log.Logger.Debugw("failed to get dmidecode uuid", "error", err)
 	}
 
-	currentMachineMetadata.OSMachineID, err = pkg_host.GetOSMachineID()
+	currentMachineMetadata.OSMachineID, err = pkghost.ReadOSMachineID()
 	if err != nil {
 		log.Logger.Warnw("failed to get os machine id", "error", err)
 	}
 
 	cctx, ccancel := context.WithTimeout(context.Background(), 20*time.Second)
-	currentSystemManufacturer, err = pkg_host.SystemManufacturer(cctx)
+	currentSystemManufacturer, err = pkghost.SystemManufacturer(cctx)
```

Comment:
Why not use CurrentSystemManufacturer?

Reply:
This `os` component will be refactored in the following PR :)

let me change here anyways